src/helpers.js

```diff
@@ -145,5 +145,9 @@ var deprecate = Marionette.deprecate = function(message, test) {
   }
 };
 
-deprecate._warn = typeof console !== 'undefined' && (console.warn || console.log) || function() {};
+deprecate._console = typeof console !== 'undefined' ? console : {};
+deprecate._console.warn = deprecate._console.warn || deprecate._console.log || function() {};
```

Comment:
that's what I'm thinking of, but still using `deprecate._console`.  The indirection is really just there for testing purposes